keyrings/codeartifact.py

```diff
@@ -1,18 +1,38 @@
 #!/usr/bin/env python
 
+import profile
```

Comment:
I'm fairly certain this isn't used anywhere. Is this import necessary?

Reply:
Woops, sorry. I should have paid closer attention. I think VSCode auto imported must have done this at some point and I didn't pick up on it. I'll strip it out.